app/services/calculators/multiple_employments_calculator.rb

```diff
@@ -4,14 +4,14 @@ class << self
       def call(submission_date)
         # multiple employments ignores inputs - hence inputs are reflected here as a set of dummy zero figures
         EmploymentIncomeCalculator::EmploymentResult.new(
-          employment: DummyEmploymentFigures.new(0, 0, 0, 0),
+          employment: DummyEmploymentFigures.new(0, 0, 0, 0, 0),
           result: EmploymentIncomeCalculator::Result.new(fixed_employment_allowance: fixed_employment_allowance(submission_date)),
         )
       end
 
     private
 
-      DummyEmploymentFigures = Data.define(:monthly_gross_income, :monthly_benefits_in_kind, :monthly_tax, :monthly_national_insurance)
+      DummyEmploymentFigures = Data.define(:monthly_gross_income, :monthly_benefits_in_kind, :monthly_tax, :monthly_national_insurance, :actively_working?)
```

Comment:
Think this should have an actively_working? method that always returns false? (not zero)